### PR TITLE
Eg 115 random

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/graphs/_GraphBD.gen.m [hunk 1]
```diff
@@ -37,6 +37,30 @@ In a binary directed (BD) graph, the edges are directed and they can be
...
+%%%% ¡id!
+GraphBD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+GraphBD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_GraphBD.gen.m]: remove this from layout

File: braph2genesis/graphs/_GraphBU.gen.m [hunk 2]
```diff
@@ -44,6 +44,30 @@ In a binary undirected (BU) graph, the edges are undirected and can be
...
+%%%% ¡id!
+GraphBU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+GraphBU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_GraphBU.gen.m]: remove this from layout

File: braph2genesis/graphs/_GraphWD.gen.m [hunk 3]
```diff
@@ -43,6 +43,36 @@ In a weighted directed (WD) graph, the edges are directed and associated
...
+%%%% ¡id!
+GraphWD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+GraphWD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_GraphWD.gen.m]: remove this from layout

File: braph2genesis/graphs/_GraphWU.gen.m [hunk 4]
```diff
@@ -50,6 +50,36 @@ In a weighted undirected (WU) graph, the edges are are undirected and associated
...
+%%%% ¡id!
+GraphWU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+GraphWU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_GraphWU.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultigraphBUD.gen.m [hunk 5]
```diff
@@ -69,6 +69,30 @@ consist of binary undirected (BU) graphs derived from the same weighted
...
+%%%% ¡id!
+MultigraphBUD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultigraphBUD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultigraphBUD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultigraphBUT.gen.m [hunk 6]
```diff
@@ -69,6 +69,30 @@ consist of binary undirected (BU) graphs derived from the same
...
+%%%% ¡id!
+MultigraphBUT.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultigraphBUT.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultigraphBUT.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerBD.gen.m [hunk 7]
```diff
@@ -51,6 +51,30 @@ In a multilayer binary directed (BD) graph, layers could have different number
...
+%%%% ¡id!
+MultilayerBD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerBD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerBD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerBU.gen.m [hunk 8]
```diff
@@ -60,6 +60,30 @@ The connectivity matrices are symmetric (within layer).
...
+%%%% ¡id!
+MultilayerBU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerBU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerBU.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerBUD.gen.m [hunk 9]
```diff
@@ -74,6 +74,36 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+%%%% ¡id!
+MultilayerBUD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerBUD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerBUD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerBUT.gen.m [hunk 10]
```diff
@@ -72,6 +72,36 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+%%%% ¡id!
+MultilayerBUT.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerBUT.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerBUT.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerWD.gen.m [hunk 11]
```diff
@@ -58,6 +58,36 @@ The connectivity matrices are symmetric (within layer).
...
+%%%% ¡id!
+MultilayerWD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerWD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerWD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultilayerWU.gen.m [hunk 12]
```diff
@@ -66,6 +66,36 @@ The connectivity matrices are symmetric (within layer).
...
+%%%% ¡id!
+MultilayerWU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerWU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultilayerWU.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexBD.gen.m [hunk 13]
```diff
@@ -51,6 +51,30 @@ In a multiplex binary directed (BD) graph, all layers have the same number
...
+%%%% ¡id!
+MultiplexBD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexBD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexBD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexBU.gen.m [hunk 14]
```diff
@@ -58,6 +58,30 @@ In a multiplex binary undirected (BU) graph, all layers have the same number
...
+%%%% ¡id!
+MultiplexBU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexBU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexBU.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexBUD.gen.m [hunk 15]
```diff
@@ -69,6 +69,30 @@ those of binary undirected (BU) multiplex graphs derived from the same
...
+%%%% ¡id!
+MultiplexBUD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexBUD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexBUD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexBUT.gen.m [hunk 16]
```diff
@@ -69,6 +69,30 @@ those of binary undirected (BU) multiplex graphs derived from the same
...
+%%%% ¡id!
+MultiplexBUT.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexBUT.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexBUT.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexWD.gen.m [hunk 17]
```diff
@@ -57,6 +57,36 @@ In a multiplex weighted directed (WD) graph, all layers have the same number
...
+%%%% ¡id!
+MultiplexWD.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexWD.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexWD.gen.m]: remove this from layout

File: braph2genesis/graphs/_MultiplexWU.gen.m [hunk 18]
```diff
@@ -64,6 +64,36 @@ In a multiplex weighted undirected (WU) graph, all layers have the same number
...
+%%%% ¡id!
+MultiplexWU.RANDOM_SEED
+%%%% ¡title!
+RANDOMIZATION SEED
+
+%%% ¡prop!
+%%%% ¡id!
+MultiplexWU.RANDOMIZATION
+%%%% ¡title!
+RANDOMIZATION VALUE
```
Comment (giovannivolpe) [inline braph2genesis/graphs/_MultiplexWU.gen.m]: remove this from layout

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
1. RANDOMIZATION should not be in the layout (typically queries never are).
2. change "RANDOMIZE ON/OFF" to "RANDOMIZATION ON/OFF" in all in the respective layout titles.
3. Verify with @mitemijalkov that all randomizations are correct that they work. 
4. Add tests for the randomizaiton in all graphs. Also in this case, verify with @mitemijalkov

**Comment by bzufiriaki** [comment]
> 3. Verify with @mitemijalkov that all randomizations are correct that they work.   **Verified**
> 4. Add tests for the randomizaiton in all graphs. Also in this case, verify with @mitemijalkov **Added randomization tests for all types of graphs. "**

**Pull request merged**